src/utils.ts

```diff
@@ -38,7 +38,8 @@ const connection_obj = {
 
 export const BitcoindClient = ({ wallet = "" }) =>
   new bitcoindClient({ ...connection_obj, wallet })
-export const bitcoindDefaultClient = BitcoindClient({ wallet: "" })
+export const bitcoindDefaultClient = BitcoindClient({ wallet: "default" }) // coinless
+export const bitcoindOutsideWalletClient = BitcoindClient({ wallet: "outside" })
```

Comment:
Minor nitpick, but we should either have `bitcoindDefaultWalletClient` and `bitcoindOutsideWalletClient` *or* `bitcoindDefaultClient` and `bitcoindOutsideClient`

Reply:
Ok for this one I wanted to be explicit about their difference. The name reflects that the default client should not be used for wallet-specific functionality (transactions). I added the "coinless" comment in there also.

Reply:
This will be addressed in a future PR more focused on tests, specifically because of the renames that also happen [here](https://github.com/GaloyMoney/galoy/blob/main/test/helpers/bitcoinCore.ts#L4).